.github/workflows/release.yaml

```diff
@@ -8,7 +8,13 @@ on:
 env:
   TAG: ${{ github.ref_name }}
   REGISTRY: ghcr.io
-  PROD_REGISTRY: ghcr.io
+  USERNAME: ${{ github.actor }}
+  PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+  ORG: rancher-sandbox
+  PROD_REGISTRY: ${{ secrets.REGISTRY_ENDPOINT }}
+  PROD_USERNAME: ${{ secrets.REGISTRY_USERNAME }}
+  PROD_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
+  PROD_ORG: rancher-sandbox
```

Comment:
I'm not sure this needed. But lets see  ;)

Reply:
Its defaulted to rancher sandbox

Reply:
This one here is just in case the prime registry destination does require some weird location. We could just change it from here, to push into the right location. It is consistent with default, yes, just wanted to have everything together in this workflow.